src/App.css

```diff
@@ -1,38 +1,272 @@
+
+html { box-sizing: border-box;}
+
+*, *:before, *:after {box-sizing: inherit}
+
+.sr-only { position: absolute; width: 1px; height: 1px; margin: -1px; border: 0; padding: 0; white-space: nowrap; clip-path: inset(100%); clip: rect(0 0 0 0); overflow: hidden;}
+
+/* Import Google Fonts */
+
+@import url('https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@300;500;700&family=Yellowtail&display=swap');
+
+/* Global Styles */
+
+body {
+  background-image: url("./assets/torontoBackground.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  color: #ffffff;
+  min-height: 100vh;
+  font-family: 'Josefin Sans', cursive;
+  font-size: 62.5%;
+  overflow-y: scroll;
+}
+
+h1 {
+  font-size: 3.4rem;
+  text-transform: uppercase;
+ }
+ 
+ h2 {
+   font-size: 2rem;
+   font-family: 'Yellowtail', cursive;
+   margin-top: -15px;
+ }
+ 
+ h3 {
+   font-size: 1.1rem;
+   text-transform: uppercase;
+   margin: 0px 0px 10px 0px;
+ }
+ 
+ p {
+   font-size: 1.1rem;
+   margin: 0;
+ }
+
+ footer {
+  position: relative;
+ }
+
+ footer p {
+  margin: 20px;
   text-align: center;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+ a {
+  color: #ffffff;
+ }
+
+ a:hover {
+  color:#12DE5F;
+  transition: 0.3s;
+ }
+ 
+ button {
+   font-size: 0.8rem;
+   font-weight: 700;
+   text-transform: uppercase;
+   background-color: #da251d;
+   color: #ffffff;
+   border: 1px solid transparent;
+   padding: 15px 32px;
+   text-align: center;
+   text-decoration: none;
+   display: inline-block;
+   margin: 20px;
+   transition: 0.3s;
+ }
+ 
+ button:hover {
+   background-color: #ffffff;
+   color: #da251d;
+ }
+
+ button:focus {
+  border: 1px solid #12DE5F;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+ img {
+  margin-top: 30px;
+ }
+
+ /* className styles */
+
+.app {
+  height: 100vh;
```

Comment:
Just noticed this might cause the white empty space at the bottom of page at smaller screen sizes.